### DISCUSSION TITLE
Weird Params in URL

### DISCUSSION THREAD

**Discussion by Mimicx** [vercel/next.js#19876] [Help]
Hey guys do you know why i am getting this  'url(&' and '&' params when i call getInitialProps function, it seems to be called 3 times with different params.

<a href="https://imgur.com/egfenE8"><img src="https://i.imgur.com/egfenE8.png" title="source: imgur.com" /></a>

this is my code snippet 

<a href="https://imgur.com/0pAL7FW"><img src="https://i.imgur.com/0pAL7FW.png" title="source: imgur.com" /></a>

**Comment by jamesmosier** [comment]
What's the url in your browser?
